include/odp/api/spec/packet_io.h

```diff
@@ -610,6 +610,8 @@ typedef union odp_pktio_set_op_t {
 		uint32_t mac_addr : 1;
 		/** Per port header offset(skip)set */
 		uint32_t skip_offset : 1;
+		/** Maximum frame length */
+		uint32_t pktio_maxlen : 1;
```

Comment:
"pktio" seems redundant here since this is inside odp_pktio_set_op_t. Should this just be the same as (or in some way aligned with) the name of the capability struct?

Reply:
The target was to align with the matching function names. I'll drop the redundant `pktio` prefix.